first_blog/config/routes.rb

```diff
@@ -0,0 +1,80 @@
+Rails.application.routes.draw do
+
+  get 'posts' => 'posts#index'
+
+  get 'posts/new', as: :new_post
+
+  get 'posts/:id' => 'posts#show', as: :post
+
+  get 'posts/:id/edit' => 'posts#edit', as: :edit_post
+
+  post 'posts' => 'posts#create'
+  
+  patch 'posts/:id' => 'posts#update'
+
+  get 'posts/:id/delete' => 'posts#delete', as: :delete_post
+
+  post 'posts/:post_id/comments' => 'comments#create', as: :post_comments
+
+  patch 'posts/:post_id/comments/:comment_id' => 'comments#update', as: :post_comment
+
+  get 'posts/:post_id/comments/:comment_id/edit' => 'comments#edit', as: :edit_comment
+
+  get 'posts/:post_id/comments/:comment_id/delete' => 'comments#delete', as: :delete_comment
```

Comment:
How does it work then?

On Monday, June 8, 2015, Brian Gates <notifications@github.com> wrote:

> In first_blog/config/routes.rb
> <https://github.com/starspace1/TIY-Assignments/pull/10#discussion_r31909664>
> :
>
> > +
> > +  get 'posts/:id/edit' => 'posts#edit', as: :edit_post
> > +
> > +  post 'posts' => 'posts#create'
> > +
> > +  patch 'posts/:id' => 'posts#update'
> > +
> > +  get 'posts/:id/delete' => 'posts#delete', as: :delete_post
> > +
> > +  post 'posts/:post_id/comments' => 'comments#create', as: :post_comments
> > +
> > +  patch 'posts/:post_id/comments/:comment_id' => 'comments#update', as: :post_comment
> > +
> > +  get 'posts/:post_id/comments/:comment_id/edit' => 'comments#edit', as: :edit_comment
> > +
> > +  get 'posts/:post_id/comments/:comment_id/delete' => 'comments#delete', as: :delete_comment
>
> that shouldn't be a GET request, bc it's destructive (same w delete_post)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/starspace1/TIY-Assignments/pull/10/files#r31909664>.
>